doc/source/new/virtualtables.rst

```diff
@@ -0,0 +1,329 @@
+                    ReadStage |            0 |                 32 |             0 |                      0 |              13 |             0
+                  Repair-Task |            0 |         2147483647 |             0 |                      0 |               0 |             0
+         RequestResponseStage |            0 |                  2 |             0 |                      0 |               0 |             0
+                      Sampler |            0 |                  1 |             0 |                      0 |               0 |             0
+     SecondaryIndexManagement |            0 |                  1 |             0 |                      0 |               0 |             0
+           ValidationExecutor |            0 |         2147483647 |             0 |                      0 |               0 |             0
+            ViewBuildExecutor |            0 |                  1 |             0 |                      0 |               0 |             0
+            ViewMutationStage |            0 |                 32 |             0 |                      0 |               0 |             0
+
+(24 rows)
+
+Internode Inbound Messaging Virtual Table
+*****************************************
+
+The ``internode_inbound``  virtual table is for the internode inbound messaging. Initially no internode inbound messaging may get listed. In addition to the address, port, datacenter and rack information includes  corrupt frames recovered, corrupt frames unrecovered, error bytes, error count, expired bytes, expired count, processed bytes, processed count, received bytes, received count, scheduled bytes, scheduled count, throttled count, throttled nanos, using bytes, using reserve bytes. A query on the ``internode_inbound`` returns following details.
+
+::
+
+ cqlsh:system_views> SELECT * FROM system_views.internode_inbound;
+ address | port | dc | rack | corrupt_frames_recovered | corrupt_frames_unrecovered | 
+ error_bytes | error_count | expired_bytes | expired_count | processed_bytes | 
+ processed_count | received_bytes | received_count | scheduled_bytes | scheduled_count | throttled_count | throttled_nanos | using_bytes | using_reserve_bytes
+ ---------+------+----+------+--------------------------+----------------------------+- 
+ ----------
+ (0 rows)
+
+SSTables Tasks Virtual Table
+****************************
+
+The ``sstable_tasks`` could be used to get information about running tasks. It lists following columns.   
+
+::
+
+ cqlsh:system_views> SELECT * FROM system_views.sstable_tasks;
+ keyspace_name | table_name | task_id | kind | progress | total | unit
+ ---------------+------------+---------+------+----------+-------+------
+
+ 
+As an example a query of ``total-progress`` gives the remaining time for a task.
+
+Other Virtual Tables
+********************
+
+Some examples of using other virtual tables are as follows.
+
+::

```

Comment:
can add comment before each of these like "Find tables with most disk usage" and "find most queried tables"

Reply:
Added. Please double check.

Reply:
while it can be used to measure read performance, its value is more in identifying which table it is thats being queried or tables with the worst latency. When we look at graphs we often see spikes in latency or count but it can be difficult to narrow down which queries it is thats causing those spikes.

Reply:
modified statement.